components/suggest/src/db.rs

```diff
@@ -1071,6 +1071,10 @@ impl<'a> SuggestDao<'a> {
             "DELETE FROM yelp_location_signs WHERE record_id = :record_id",
             named_params! { ":record_id": record_id.as_str() },
         )?;
+        self.conn.execute_cached(
+            "DELETE FROM yelp_custom_details WHERE record_id = :record_id",
+            named_params! { ":record_id": record_id.as_str() },
+        )?;
```

Comment:
Does the icon data in `icons` need to be deleted too? Does that happen somewhere else?

Reply:
Yup, icon deletion happens [here](https://github.com/mozilla/application-services/blob/a757c1ec4083259622e366ea4ce5a2c2189b2088/components/suggest/src/db.rs#L167).